tools/pythonpkg/src/include/duckdb_python/pyrelation.hpp

```diff
@@ -100,7 +100,7 @@ struct DuckDBPyRelation {
 
 	unique_ptr<DuckDBPyRelation> Map(py::function fun);
 
-	unique_ptr<DuckDBPyRelation> Join(DuckDBPyRelation *other, const string &condition);
+	unique_ptr<DuckDBPyRelation> Join(DuckDBPyRelation *other, const string &condition, const string &type);
```

Comment:
What do you think about naming this parameter `how` instead of `type` for two reasons?

1. `type` is already a built-in in Python, which will make some linters complain.
2. `how` is the parameter used by pandas in [`merge()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.merge.html) and [`join()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.join.html) so most developers will already be familiar with it.

Reply:
Sure